<macports>Patch from debian: http://patch-tracker.debian.org/package/eperl/2.2.14-16</macports>
Patch preprocessor to work as documented wrt comments. (Closes: #360891)
Adds also a hint that now this is also a solution to #360790.

Index: eperl/eperl_pp.c
===================================================================
--- eperl.orig/eperl_pp.c	2010-06-06 02:09:25.000000000 +0200
+++ eperl/eperl_pp.c	2010-06-06 02:09:25.000000000 +0200
@@ -395,7 +395,9 @@
 
                 /* create replacement string: just a newline 
                  * to preserve line numbers */
-                sprintf(caStr, "\n");
+		/* No, don't do that, the documentation says otherwise.
+		 * See also http://bugs.debian.org/360891 */
+                sprintf(caStr, "");
                 cp = caStr;
             }
 
Index: eperl/eperl.pod
===================================================================
--- eperl.orig/eperl.pod	2010-06-06 02:16:48.000000000 +0200
+++ eperl/eperl.pod	2010-06-06 02:17:46.000000000 +0200
@@ -335,6 +335,13 @@
   #!/bin/sh
   echo quux
 
+Alternatively you can add a preprocessor comment in the first line,
+e.g. like this:
+
+  #c This is a comment to preserve the shebang line in the following line
+  #!/bin/sh
+  echo <: print "quux"; :>
+
 And finally you can use ePerl directly from within Perl programs by the use of
 the Parse::ePerl(3) package (assuming that you have installed this also; see
 file F<INSTALL> inside the ePerl distribution for more details):
